app/views/games/_table.html.erb

```diff
@@ -10,20 +10,19 @@
 <% else %>
   <table class="game-table table table-bordered table-striped">
     <tr>
-      <th>Name</th>
-      <% if show_console %>
-        <th>Console</th>
-      <% end %>
+      <th>Top Image</th>
+      <th width='80%'>Name</th>
     </tr>
     <% @matches.each do |game| %>
       <tr>
+        <td>
+          <%= game.pictures.empty? ? 'No image!' : link_to(image_tag(game.top_rated_picture.image_url, class: 'table-image'), game) %>
```

Comment:
The text `No image!` looks kind of janky, could we do a placeholder image instead?